show/muxcable.py

```diff
@@ -419,3 +427,97 @@ def cableinfo(port):
 
     body = [[vendor, part_num]]
     click.echo(tabulate(body, headers=headers))
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hardware mode on a port"""
+    pass
+
+
+@hwmode.command()
+@click.argument('port', metavar='<port_name>', required=False, default=None)
+def mux_direction(port):
+    if port is not None:
+
+        """Show muxcable summary information"""
+        if platform_sfputil is not None:
+            physical_port_list = platform_sfputil_helper.logical_port_name_to_physical_port_list(port)
+
+        if not isinstance(physical_port_list, list):
+            click.echo("ERR: Unable to get a port on muxcable port")
+            sys.exit(EXIT_FAIL)
+            if len(physical_port_list) != 1:
+                click.echo("ERR: Unable to get a single port on muxcable")
```

Comment:
Message isn't very clear. Maybe `"ERR: Found multiple physical ports associated with {}".format(port)`? And maybe even print out the list of physical ports to aid in debugging?

Reply:
fixed